README.md

```diff
@@ -37,33 +37,33 @@ The Flow Emulator can be run in different modes and settings, all of them are de
+| `--persist`                   | `FLOW_PERSIST`                | false          | Enable persistence of the state between restarts                                                                                                                                                                                                    |
+| `--dbpath`                    | `FLOW_DBPATH`                 | `./flowdb`     | Specify path for the database file persisting the state                                                                                                                                                                                             |
+| `--simple-addresses`          | `FLOW_SIMPLEADDRESSES`        | `false`        | Use sequential addresses starting with `0x1`                                                                                                                                                                                                        |
+| `--token-supply`              | `FLOW_TOKENSUPPLY`            | `1000000000.0` | Initial FLOW token supply                                                                                                                                                                                                                           |
+| `--transaction-expiry`        | `FLOW_TRANSACTIONEXPIRY`      | `10`           | [Transaction expiry](https://docs.onflow.org/flow-go-sdk/building-transactions/#reference-block), measured in blocks                                                                                                                                |
+| `--storage-limit`             | `FLOW_STORAGELIMITENABLED`    | `true`         | Enable [account storage limit](https://docs.onflow.org/cadence/language/accounts/#storage-limit)                                                                                                                                                    |
+| `--storage-per-flow`          | `FLOW_STORAGEMBPERFLOW`       |                | Specify size of the storage in MB for each FLOW in account balance. Default value from the flow-go                                                                                                                                                  |
+| `--min-account-balance`       | `FLOW_MINIMUMACCOUNTBALANCE`  |                | Specify minimum balance the account must have. Default value from the flow-go                                                                                                                                                                       |
+| `--transaction-fees`          | `FLOW_TRANSACTIONFEESENABLED` | `false`        | Enable variable transaction fees and execution effort metering <br> as decribed in [Variable Transaction Fees: Execution Effort](https://github.com/onflow/flow/pull/753)   FLIP                                                                    |
```

Comment:
we could use this flag in the future to pass non-default configured fees in a JSON format maybe.